rn/Teacher/ios/TeacherUITests/Files/FilesUITests.swift

```diff
@@ -156,6 +156,10 @@ class FilesUITests: MiniCanvasUITestCase {
         FileEditItem.delete.tap()
         app.find(label: "Delete").tap()
 
+        // FIXME: These should have happened automatically.
```

Comment:
Is there a ticket for this?

Reply:
https://instructure.atlassian.net/browse/MBL-14496